### ISSUE TITLE
Up/down arrow keys should move selection to cell above below

### ISSUE THREAD

**Issue by pjasiun** [ckeditor/ckeditor5-table#108]
I wanted to fill the table column and I was surprised that when I press down key selection moved to the cell on the right, not below. The same for arrow up - selection should be moved to the cell above, not on the left. The current behavior makes filling column very inconvenient.

**Comment by jodator** [comment]
I AFAICS we left it to the browsers. But also for me the expected behavior is such that using arrows would allow to navigate up/down visually as it does on Chrome.

ps.: it works as described in Firefox

**Comment by Reinmar** [comment]
Yep, for now we do nothing and let the browser decide. We can change it at any time.

**Comment by Reinmar** [comment]
Just so you know – this is a tricky issue. Pressing down arrow in a cell might move the selection to another line of text in the same cell or may need to move it to another cell. We may look for some hacks though, like handling the arrow keys only if the selection is at the very beginning/end of a cell. That'd handle empty cells as wells as non-empty ones but with an additional step of first moving your caret to the very end of the content and then moving to the next cell.

**Comment by Reinmar** [comment]
Another case: https://github.com/ckeditor/ckeditor5-table/issues/175
